abi/abi.go

```diff
@@ -267,6 +277,7 @@ func Encode(it Item) []byte {
 
 // Decodes ABI encoded bytes into an [Item] according to
 // the 'schema' defined by t. For example:
+//
```

Comment:
i think my `go fmt` added this, i can remove from the diff if you want

Reply:
Let's remove it. I wonder what go fmt setting is doing this. I also have go fmt and it doesn't make this change.

Reply:
done